### PR TITLE
Merge 19.1 code freeze

### PR DESCRIPTION
**Pull request by mokagio**

### CODE REVIEW

File: WordPress/Resources/release_notes.txt [hunk 1]
```diff
@@ -1,12 +1,20 @@
-What time is it? Our brand-new time zone selection screen knows. Get a suggested time zone based on your device and improve search while you’re at it.
+* [*] Signup: Fixed bug where username selection screen could be pushed twice. [#17624]
+* [**] Threaded comments: comments can now be moderated via a drop-down menu on each comment. [#17758]
+* [**] Reader post details Comments snippet: added ability to manage conversation subscription and notifications. [#17749]
```
Comment (mokagio) [inline WordPress/Resources/release_notes.txt:3]: Notice that this item regarding the Reader is only part of the WordPress release notes 👀

**Review by mokagio — COMMENTED** [review]
- [x] Switch to production Pods
- [x] New version header in `RELEASE-NOTE.txt` 
- [x] `Localizable.strings` updated 
- [x] `release_notes.txt` updated with notes from `RELEASE-NOTE.txt` for current version 
- [x] Version update in `.xcconfig` and `download_metadata.swift`

**Comment by AliSoftware** [comment]
Unit tests failing on:
```
WKCookieJarTests.testGetCookies()
		WKCookieJarTests.testHasCookieMatching()
		WKCookieJarTests.testRemoveCookies()
```
I think those are known to be flaky tho? So maybe re-running them will fix it?

File: WordPress/Jetpack/Resources/release_notes.txt [hunk 2]
```diff
@@ -1,12 +1,19 @@
-What time is it? Our brand-new time zone selection screen knows. Get a suggested time zone based on your device and improve search while you’re at it.
+* [*] Signup: Fixed bug where username selection screen could be pushed twice. [#17624]
```
Comment (AliSoftware) [inline WordPress/Jetpack/Resources/release_notes.txt]: Just double-checking, given that everything login-related in Jetpack might not always be applicable…

I'm often confusing Signup and Signin but IINM Signup is about account creation (not logging in using existing account), which is something that's not yet applicable to Jetpack? Except if this is related to the recent addition to SIWA support in Jetpack (which can lead to account creation in that case)?

cc @leandroalonso

File: WordPress/Jetpack/Resources/release_notes.txt [hunk 3]
```diff
@@ -1,12 +1,19 @@
...
+* [**] Threaded comments: comments can now be moderated via a drop-down menu on each comment. [#17758]
+* [**] Accessibility: VoiceOver and Dynamic Type improvements on Activity Log and Schedule Post calendars [#17756, #17761, #17780]
+* [*] Weekly Roundup: Fix a crash which was preventing weekly roundup notifications from appearing [#17765]
+* [*] Self-hosted login: Improved error messages. [#17724]
```
Comment (AliSoftware) [inline WordPress/Jetpack/Resources/release_notes.txt]: Is that applicable to Jetpack, given it's about Self-hosted? \cc @leandroalonso

File: WordPress/Jetpack/Resources/release_notes.txt [hunk 4]
```diff
@@ -1,12 +1,19 @@
...
+* [*] Block editor: Hide help button from Unsupported Block Editor. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4352]
+* [*] Block editor: Add contrast checker to text-based blocks [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4357]
+* [*] Block editor: Fix missing Featured Image translations [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4464]
+* [*] Block editor: Fix missing translations of color settings [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4479]
+* [*] Block editor: Fix cut-off setting labels by properly wrapping the text [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4475]
+* [*] Block editor: Highlight text: fix applying formatting for non-selected text [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4471]
+* [**] Block editor: Fix Android handling of Hebrew and Indonesian translations [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4397]
+* [***] Self-hosted sites: Fixed a crash when saving media and no Internet connection was available. [#17759]
```
Comment (AliSoftware) [inline WordPress/Jetpack/Resources/release_notes.txt]: Same question as above, re: self-hosted vs Jetpack

File: WordPress/Jetpack/Resources/release_notes.txt [hunk 5]
```diff
@@ -1,12 +1,19 @@
...
+* [*] Publicize: Fixed an issue where a successful login was not automatically detected when connecting a Facebook account to Publicize. [#17803]
```
Comment (AliSoftware) [inline WordPress/Jetpack/Resources/release_notes.txt:15]: Not sure about this one: is Facebook connect available to Jetpack? I think so, but better get confirmation from @leandroalonso on that one too while we're at it.

**Review by AliSoftware — COMMENTED** [review]
Overall looks good, but I'm unsure about multiple items in the Jetpack Release Notes being applicable to Jetpack…

File: WordPress/Jetpack/Resources/release_notes.txt (hunk 2 above)
Comment (leandroalonso) [reply WordPress/Jetpack/Resources/release_notes.txt]: > Except if this is related to the recent addition to SIWA support in Jetpack (which can lead to account creation in that case)?

Yes, you're right. Although I'd consider any account creation not part of Jetpack, given if a user creates an account there they can't really do much (site creation is not enabled).

File: WordPress/Jetpack/Resources/release_notes.txt (hunk 3 above)
Comment (leandroalonso) [reply WordPress/Jetpack/Resources/release_notes.txt]: No, not applicable. Self-hosted is not supported in the Jetpack app.

File: WordPress/Jetpack/Resources/release_notes.txt (hunk 4 above)
Comment (leandroalonso) [reply WordPress/Jetpack/Resources/release_notes.txt]: Same as before.

File: WordPress/Jetpack/Resources/release_notes.txt (hunk 5 above)
Comment (leandroalonso) [reply WordPress/Jetpack/Resources/release_notes.txt:15]: Yes, Publicize is available on Jetpack. Publicize is actually a Jetpack feature (available on .com too): https://jetpack.com/support/publicize/

File: WordPress/Jetpack/Resources/release_notes.txt (hunk 2 above)
Comment (AliSoftware) [reply WordPress/Jetpack/Resources/release_notes.txt]: Right, that's what I thought.

Given that's "not really part of Jetpack" (even if _technically_ they can create an account via SIWA, it's mostly a side effect of SIWA support but they can't really do much anyway), should we really include this into the Release Notes then? Or would it make more sense to remove it?

Comment (leandroalonso) [reply WordPress/Jetpack/Resources/release_notes.txt]: Let's remove it. :)

Comment (mokagio) [reply WordPress/Jetpack/Resources/release_notes.txt]:
```suggestion
```

File: WordPress/Jetpack/Resources/release_notes.txt (hunk 3 above)
Comment (mokagio) [reply WordPress/Jetpack/Resources/release_notes.txt]:
```suggestion
```

Comment (mokagio) [reply WordPress/Jetpack/Resources/release_notes.txt]: Thank you for this. Unlike the other note, which I altogether missed, I did "try" this one on device. The "Enter your existing side address" from the prologue screen got me.


As an aside, when the app asks for the address, my phone suggests my home address 🙃 I haven't looked into whether that's an iOS thing or something we can configure, but thought it funny:

IMG_1D3EC5EC1398-1

File: WordPress/Jetpack/Resources/release_notes.txt (hunk 4 above)
Comment (mokagio) [reply WordPress/Jetpack/Resources/release_notes.txt]:
```suggestion
```

**Review by mokagio — COMMENTED** [review]
Thanks @AliSoftware and @leandroalonso. I've suggested removals of the lines you highlighted and will commit them in one go via the GitHub UI.

**Review by AliSoftware — APPROVED** [review]

**Pull request merged**